atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/CachingTransaction.java

```diff
@@ -114,7 +118,11 @@ public Transaction delegate() {
     @Override
     public Map<Cell, byte[]> get(TableReference tableRef, Set<Cell> cells) {
         if (cells.isEmpty()) {
-            log.info("Attempted get on '{}' table with empty cells argument", tableRef);
+            if (log.isTraceEnabled()) {
+                log.trace(String.format("Attempted get on %s table with empty cells argument", tableRef), new Exception());
```

Comment:
String.format isn't necessary, the logger knows what to do with params + throwable:
`log.trace("Attempted get on {} table with empty cells argument", tableRef, new Exception())`